online-inference/stable-diffusion/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG CUDA_RELEASE=11.6.2-cudnn8-devel-ubuntu20.04
+FROM nvidia/cuda:${CUDA_RELEASE} AS base
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt upgrade -y
+RUN apt update && apt install -y python3 python3-pip
+
+FROM base as isvc
+RUN mkdir -p /app
+ADD service/ /app/
+WORKDIR /app
+RUN pip3 install --upgrade pip
```

Comment:
Combine as many steps so that we don't have so many snapshots along the way.  pip3 should also be used with no cache.